src/molecules/lists/unordered-list.tsx

```diff
@@ -2,6 +2,12 @@ import React from "react";
 import { Icons } from "../../atoms/constants/icons";
 import { StringUtils } from "andculturecode-javascript-core";
 
+// -------------------------------------------------------------------------------------------------
+// #region Constants
+// -------------------------------------------------------------------------------------------------
+
+export const UnorderedListIconClassName = "-has-icon";
```

Comment:
Looks like this is missing an endregion tag

Reply:
@brandongregoryscott Thanks for catching that. I added the endregion tag 👍